### PR TITLE
Support for request params and forward slash

### PR DESCRIPTION
**Pull request by naveenkewalramani**

### CODE REVIEW

File: lib/configure_semian/net_http.rb [hunk 1]
```diff
@@ -12,14 +12,23 @@ def request(req, body = nil, &block)
...
       end
       specs ||= ConfigureSemian::SemianConfiguration.service_configs[:default]
       timeout = specs[:timeout] || specs['timeout']
       return timeout
     end
+
+    private
+    def process_request_path(path)
+      path = path.split('?').first
+      path.gsub('//',"\/")
```
Comment (rakeshkumariiita) [inline lib/configure_semian/net_http.rb]: check for regex for /+ type to replace all.

File: lib/configure_semian/net_http.rb [hunk 2]
```diff
@@ -12,14 +12,23 @@ def request(req, body = nil, &block)
     end
 
     def get_request_timeout_value(host, path)
+      path = process_request_path(path)
       if !ConfigureSemian::SemianConfiguration.service_configs[host.intern].nil?
         specs = ConfigureSemian::SemianConfiguration.service_configs[host.intern][path.intern]
+        specs ||= ConfigureSemian::SemianConfiguration.service_configs[host.intern]["/#{path}".intern]
+        specs ||= ConfigureSemian::SemianConfiguration.service_configs[host.intern][path[1..-1].intern]
```
Comment (rakeshkumariiita) [inline lib/configure_semian/net_http.rb]: use ^/ as it will detect start of character for it.

File: lib/configure_semian/net_http.rb (hunk 1 above)
Comment (naveenkewalramani) [reply lib/configure_semian/net_http.rb]: done

File: lib/configure_semian/net_http.rb (hunk 2 above)
Comment (naveenkewalramani) [reply lib/configure_semian/net_http.rb]: replaced with sub

**Pull request merged**